src/key_extractor.rs

```diff
@@ -17,6 +18,19 @@ pub trait KeyExtractor: Clone {
     /// Extraction method
     fn extract(&self, req: &ServiceRequest) -> Result<Self::Key, Self::KeyExtractionError>;
 
+    /// The content you want to show it when the rate limit is exceeded.<br>
+    /// The [`NotUntil`] will be passed to it and it has enough information.<br>
```

Comment:
Is there a reason why you use inline HTML? I personally don't like the `<br>` tags here.